.eslintrc.js

```diff
@@ -18,11 +18,9 @@ module.exports = {
   // add your custom rules here
   'rules': {
     // don't require .vue extension when importing
-    'import/extensions': ['error', 'always', {
-      'js': 'never',
-      'vue': 'never'
-    }],
+    'import/extensions': false,
```

Comment:
What is the reason for this change?

Reply:
its like that for now, my IDE had some issues as it didnt recognize the exceptions